packages/hooks/src/useTotalAccountBalance.ts

```diff
@@ -1,21 +1,25 @@
 import useRealTotalAssetCurrency from './useTotalAssetCurrency';
 import useExchangeRate from './useExchangeRate';
-
 /**
  * we can use this hook to get the total balance of the given accounts list.
  * it loops through the accounts list and adds the balance of each account
  * to the total balance, it also converts the balance to the currency of the
  * first account in the list
  */
+type TAccounts = {
```

Comment:
TUseTotalAccountBalance. TAccounts is irrelevant and my be confused with our other types which are related to our accounts object. choose your type naming based on the name of the components or functions